allennlp/training/optimizers.py

```diff
@@ -278,6 +278,11 @@ def __init__(
                     " Alternatively, you can remove this optimizer from the provided `optimizers`"
                     " if it is not relevant to a particular parameter group."
                 )
+        # default optimizer is required, but may not be used
```

Comment:
"must not be used" or "using it is optional"?

Reply:
I know what you mean from context, but it would be easier to read if worded unambiguously.